test/metabase/query_processor_test/timezones_test.clj

```diff
@@ -113,56 +113,57 @@
-                                             (field-identifier :users :last_login)
-                                             (hx/raw "{{date1}}")
-                                             (hx/raw "{{date2}}")]
-                                  :order-by [[(field-identifier :users :id) :asc]]})
-                 :template-tags {:date1 {:name "date1" :display_name "Date1" :type "date"}
-                                 :date2 {:name "date2" :display_name "Date2" :type "date"}}}
-    :parameters [{:type   "date/single"
-                  :target ["variable" ["template-tag" "date1"]]
-                  :value  "2014-08-02T02:00:00.000000"}
-                 {:type   "date/single"
-                  :target ["variable" ["template-tag" "date2"]]
-                  :value  "2014-08-02T06:00:00.000000"}]}
-
-   "field filter w/ date range"
-   {:native     {:query         (honeysql->sql
-                                 {:select   (mapv (partial field-identifier :users)
-                                                  [:id :name :last_login])
-                                  :from     [(table-identifier :users)]
-                                  :where    (hx/raw "{{ts_range}}")
-                                  :order-by [[(field-identifier :users :id) :asc]]})
-                 :template-tags {:ts_range {:name         "ts_range"
-                                            :display_name "Timestamp Range"
-                                            :type         "dimension"
-                                            :widget-type  :date/all-options
-                                            :dimension    [:field (mt/id :users :last_login) nil]}}}
-    :parameters [{:type   "date/range"
-                  :target ["dimension" ["template-tag" "ts_range"]]
-                  :value  "2014-08-02~2014-08-03"}]}
-
-   "field filter w/ single date"
-   {:native     {:query         (honeysql->sql
-                                 {:select   (mapv (partial field-identifier :users)
-                                                  [:id :name :last_login])
-                                  :from     [(table-identifier :users)]
-                                  :where    (hx/raw "{{just_a_date}}")
-                                  :order-by [[(field-identifier :users :id) :asc]]})
-                 :template-tags {:just_a_date {:name         "just_a_date"
-                                               :display_name "Just A Date"
-                                               :type         "dimension"
-                                               :widget-type  :date/all-options
-                                               :dimension    [:field (mt/id :users :last_login) nil]}}}
-    :parameters [{:type   "date/single"
-                  :target ["dimension" ["template-tag" "just_a_date"]]
-                  :value  "2014-08-02"}]}})
+  (binding [hx/*honey-sql-version* (sql.qp/honey-sql-version driver/*driver*)]
```

Comment:
Only change here is to wrap it in `binding [hx/*honey-sql-version* (sql.qp/honey-sql-version driver/*driver*)] ...)`